src/main/kotlin/io/polygon/kotlin/sdk/rest/Aggregates.kt

```diff
@@ -2,6 +2,7 @@ package io.polygon.kotlin.sdk.rest
 
 import com.thinkinglogic.builder.annotation.Builder
 import com.thinkinglogic.builder.annotation.DefaultValue
+import io.ktor.http.*
```

Comment:
Seeing this import added into a lot of files where it doesn't seem necessary, is there a reason I'm not thinking of?

Reply:
It's to make io.ktor.http.URLBuilderKt#path resolvable.
To be honest I don't fully understand why it wasn't needed in previous ktor version, but they changed something how the function is declared, and it stopped compiling without explicit import.

If you can suggest a different preferred way how to use io.ktor.http.URLBuilderKt#path I will update these places.

Reply:
@mmoghaddam385 I could change those to `import io.ktor.http.path` to make it more explicit, other than that out of ideas how to avoid the import

Reply:
Either way is fine, we can leave it as is no problem. I just wanted to make sure this wasn't an accidental addition or superfluous.

Reply:
Thanks, got it, let's keep it as is then .